app/helpers/feeder_form_builder.rb

```diff
@@ -50,14 +53,7 @@ def date_field(method, options = {})
 
   def time_field(method, options = {})
     value = options[:value] || object&.public_send(method)
-
-    # end timestamps are fudged 1-minute to look inclusive
-    if options[:fudge] && value && value == value.beginning_of_day
```

Comment:
Removed all the code around "fudging" timestamps by 1 second.  That was an Augury end_date thing.